src/clients/BundleDataClient/BundleDataClient.ts

```diff
@@ -899,24 +894,18 @@ export class BundleDataClient {
                 );
                 // At this point, the v3RelayHashes entry already existed meaning that there is a matching deposit,
                 // so this fill is validated.
-                v3RelayHashes[relayDataHash].fill = fill;
```

Comment:
This diff might be removable if we change `updatedFill` to `fill` above and change the `async (fill)` to `async (_fill)`. This way we again only need to deal with `fill` after the `verifyFillRepayment` call

Reply:
https://github.com/across-protocol/sdk/pull/842/commits/b57ac22cf3bff05267008a4460b0b981e46ff510